backend/app/services/MetricsService.scala

```diff
@@ -0,0 +1,76 @@
+package services
+
+import com.amazonaws.auth.AWSCredentialsProvider
+import com.amazonaws.services.cloudwatch.{AmazonCloudWatch, AmazonCloudWatchClientBuilder}
+import com.amazonaws.services.cloudwatch.model.{Dimension, MetricDatum, PutMetricDataRequest}
+import utils.{AwsCredentials, Logging}
+
+import java.util.Date
+import scala.util.control.NonFatal
+
+case class MetricUpdate(name: String, value: Double)
+
+object Metrics {
+  val namespace = "PFI"
+
+  val itemsIngested = "IngestedItems"
+  val itemsFailed = "IngestItemsFailed"
+  val batchesIngested = "IngestBatchesIngested"
+  val batchesFailed = "IngestBatchesFailed"
+  val failureToResultMapper = "ErrorsInGiantFailureToResultMapper"
+
+
+  def metricDatum(name: String, dimensions: List[Dimension], value: Double): MetricDatum = {
+    new MetricDatum()
+      .withMetricName(name)
+      .withTimestamp(new Date())
+      .withDimensions(dimensions: _*)
+      .withValue(value)
+  }
+}
+
+trait MetricsService {
+  def updateCloudwatchMetrics(metrics:List[MetricUpdate]): Unit
+  def updateCloudwatchMetric(metricName: String, metricValue: Double = 1): Unit
```

Comment:
Can these be `updateMetric` and `updateMetrics` - I don't think it makes sense to refer to a specific implementation of a metric service from the trait.